.github/workflows/ci.yml

```diff
@@ -86,6 +89,7 @@ jobs:
           tools/ci/install_byond.sh
           source $HOME/BYOND/byond/bin/byondsetup
           tools/ci/dm.sh -DCIBUILDING paradise.dme
+          echo '${{ matrix.maptype }}' > data/next_map.txt
```

Comment:
I want to take a moment to appreciate how a mere 4 lines makes the CI a fuck ton more useful